pages/index.js

```diff
@@ -1,11 +1,8 @@
-import Tag from "../components/ui/Tag";
-
+import Link from "../components/UI/Link";
```

Comment:
It looks good, but you have to change The way you are calling The component , The UI should equal to Ui.